### PR TITLE
Feature/sch 568: Customized skype sign-in 

### PR DESCRIPTION
**Pull request by harshvshah**

### CODE REVIEW

**Review by alexknipfer — APPROVED** [review]

File: Node/src/dialogs.ts [hunk 1]
```diff
@@ -111,7 +112,7 @@ export class AuthDialog extends builder.Dialog {
                 msg = new builder.Message(session)
                     .attachments([
                         new builder.SigninCard(session)
-                            .text("connect_prompt")
+                        .text(args.providerId === 'azuread-openidconnect' ? args.skypeSignIn : "connect_prompt")
```
Comment (carblue5757) [inline Node/src/dialogs.ts]: I dont know that you can just call args.providerId.   Try doing what they did above for the buttonUrl -- `opt.buttonUrl`

Comment (carblue5757) [reply Node/src/dialogs.ts]: same for opt.skypeSignIn

File: Node/src/index.ts [hunk 2]
```diff
@@ -342,14 +343,20 @@ export class BotAuthenticator {
...
                         res.end();
                     } else {
-                        if (this.options.successRedirect) {
+                        if (this.options.successRedirect && providerId === "azuread-openidconnect") {
+                            res.status(302);
+                            res.header("Location", `${options.successRedirect}?providerId=${providerId}#${encodeURIComponent(magic)}`);
+                            res.send("redirecting...");
+                            res.end();
+                        }
+                        else if (this.options.successRedirect) {
```
Comment (carblue5757) [inline Node/src/index.ts]: move the `else if` up one line, to be on the same line as the `}`

File: Node/src/index.ts [hunk 3]
```diff
@@ -342,14 +343,20 @@ export class BotAuthenticator {
...
                             res.status(302);
                             res.header("Location", `${options.successRedirect}#${encodeURIComponent(magic)}`);
                             res.send("redirecting...");
                             res.end();
-                        } else {
-                            // show the user the magic number they need to enter in the chat window
-                            res.end(`You're almost done. To complete your authentication, put "${ magic }" in our chat.`);
+                        }
+                        else {
```
Comment (carblue5757) [inline Node/src/index.ts]: move the else up one line.

**Review by carblue5757 — CHANGES_REQUESTED** [review]

**Review by carblue5757 — APPROVED** [review]

**Pull request merged**